en/db-migrations.md

```diff
@@ -131,3 +136,25 @@ $migration::run([
 | --verbose | Output of debugging information during operation (Running only)
 | --no-auto-increment | Disable auto increment (Generating only)
 | --help | Shows this help
+
+## Timestamp based migrations
+
+Current approach is useful when more than one developer is participating in Database Structure management.
+Use `'migrationsTsBased' => true` in config file or `--ts-based` option in CLI environment.
+Also, you need to specify suffix `descr`, which could be anything you want, for example: semantic version.
+
+Current command
+```
+vendor/bin/phalcon-migrations generate --ts-based --descr=1.0.0
+```
+
+Will produce folder name with such names
+
+* 1582539287636860_1.0.0
+* 1682539471102635_1.0.0
+* 1782539471102635_1.0.0
+
+Migrations will be executed from oldest to newest.
+
+> **NOTE**: In case when one of past migrations wasn't executed and was introduced, after next run it executes without any problem.
```

Comment:
> In case when one of past migrations wasn't executed and was introduced, after next run it executes without any problem.

What do you mean by this? Old migration that was not executed will be executed with the new run?

Reply:
yes, imagine when one of 2 migrations was executed firstly, because it was merged earlier. So instead of checking only migrations that are after the last one which was executed, it scans also the previous.

Reply:
In that case:

Whenever migrations are run, the application scans all available migrations and their status irrespective of their "age". If one or more were not executed in a previous run, they will be executed in the next run.